packages/components/src/utils/render-queue.js

```diff
@@ -45,6 +46,17 @@ class RenderQueue {
       element.render(); // eslint-disable-line no-underscore-dangle
     });
   }
+
+  clearQueue() {
+    while (this.queue.size > 0) {
```

Comment:
this is so that our tests render correctly, and also to prevent a flicker when we have disabled options in our list menu. However this could unintentionally introduce an infinite loop. Currently this is not the case, and I don't see how this would currently happen, but I wanted to highlight this piece of code so that it does not go unnoticed.